### DISCUSSION TITLE
Confused as to how to interpret linkage plots?

### DISCUSSION THREAD

**Discussion by ahdee** [stuart-lab/signac#1460] [Q&A]
First of all, thanks for the awesome package! 

I'm confuse as to how to interpret the peak-to-gene plots. In figure 2I for example it looks like this. 

![image](https://github.com/stuart-lab/signac/assets/9808122/120f1a26-058b-4290-9403-0a62b27aa74a)

I think peak-to-gene its basically the correlation of the atacseq peaks (openess) to expression? which makes sense but I have a few questions. 

1. what do the archs mean, for example from the figure it looks like some of the exons are connected to each other - does this imply that they exons that are connected are also also correlated in peaks? 
2. does each exon that have an arch mean there was a significant relationship between expression and atac peaks? Moroever what do the scores mean? are they just the magnitude of the correlations? 
3. what if there are no archs. For example I have a gene that looks like this. 

![image](https://github.com/stuart-lab/signac/assets/9808122/cad3fa3c-123c-4ea7-af55-13c6e815c8ab)

4. what if there is an arch but it does not connect to any other exons? 

![image](https://github.com/stuart-lab/signac/assets/9808122/1e637d78-c54f-4ca4-b394-dd3932f30f38)

The above were generated with the CoveragePlot function. 

thanks in advance! much appreciated.

**Comment by timoast** [comment]
> what do the archs mean, for example from the figure it looks like some of the exons are connected to each other - does this imply that they exons that are connected are also also correlated in peaks?

The arcs connect a peak to the TSS of the gene that it is correlated with, they don't connect exons

> does each exon that have an arch mean there was a significant relationship between expression and atac peaks? Moroever what do the scores mean? are they just the magnitude of the correlations?

Please see the documentation for `LinkPeaks()` and the description in the Signac paper.

> what if there are no archs. For example I have a gene that looks like this.

Either you haven't run LinkPeaks, or you ran it for a subset of genes that doesn't include that gene, or there were no correlated peaks identified for that gene
